library/build.gradle

```diff
@@ -26,8 +29,17 @@ android {
     useLibrary 'org.apache.http.legacy'
 }
 
+dependencies {
+    lp2Implementation files('../../shared/framework_all.jar')
+    lp3Implementation files('../../shared/lp3_framework_all.jar')
```

Comment:
where do we have this lp3 file? I need to add it to the fota-updater app

Reply:
Dumb question! It's in the LightOS folder in your new PR - I'll take it from there